### PR TITLE
Add TodoList, Todo component

### PR DESCRIPTION
**Pull request by riseshia**
connect to https://github.com/riseshia/gannbaruzoi/issues/40

## Todo

- [x] checkbox for done/undone
- [x] delete
- [x] input field for new todo

### CODE REVIEW

**Comment by marocchino** [comment]
Is it work in progress?

**Comment by riseshia** [comment]
yep. I will request review when it done XD

File: src/components/Todo.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</template>
+
+<script>
+import { Vue, Component, Prop, p, Watch } from "av-ts"
+
+@Component({
+  name: "todo"
+})
+class Todo extends Vue {
+  @Prop todoId        = p({ type: Number, required: true })
```
Comment (marocchino) [inline src/components/Todo.vue]: (maybe)
PersonalIy, don't like this alignment. cause It broke commit log.
But, If you want it. Keep do it.
https://medium.com/@fagnerbrack/why-aligning-statements-will-haunt-you-c7385a3b24d#.5m33lrazs

File: test/unit/specs/Todo.spec.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  it("renders currect contents", () => {
+    const vm = new Todo({
+      propsData: {
+        todoId: 1, description: "Todo 1", type: "ROOT",
+        estimatedSize: 2, status: false
+      }
+    }).$mount();
+
+    const actual = vm.$el.textContent;
+    assert.match(actual, /Todo 1/);
```
Comment (marocchino) [inline test/unit/specs/Todo.spec.ts]: (maybe) is not it unnecessary regexp?
```
assert.include(actual, 'Todo 1');
```

**Review by marocchino — APPROVED** [review]

File: test/unit/specs/Todo.spec.ts (hunk 2 above)
Comment (riseshia) [reply test/unit/specs/Todo.spec.ts]: OTL I missed it. I misunderstood it's for collection
http://chaijs.com/api/assert/#method_include

File: src/components/Todo.vue (hunk 1 above)
Comment (riseshia) [reply src/components/Todo.vue]: agree. I will update XD

**Pull request merged**